src/service.rs

```diff
@@ -202,13 +237,24 @@ where
 		client.clone(),
 	);
 
+	// TODO: is it bad if we create these when we're not in a runtime that uses EVM?
```

Comment:
As far as I can tell this doesn't cause any issues when it's instantiated on a node without frontier set up in other ways.

It might create a frontier DB file? I need to confirm that before we merge

Reply:
Does this open/create a db under the same path as the parachain one? If so, I would maybe tend to go a route where this is another path with a clear prefix maybe?

Reply:
It creates a separate `frontier/db` next to the main `db` folder:

```
branan@branan-laptop:~/proj/centrifuge/centrifuge-chain$ tree -d /tmp/centrifuge-chain/parachain/2000/data/chains/devel_local/
/tmp/centrifuge-chain/parachain/2000/data/chains/devel_local/
├── db
│   └── full
├── frontier
│   └── db
├── keystore
└── network

6 directories
```